data/assets/07_matrix_view.css

```diff
@@ -1,3 +1,10 @@
+:root {
```

Comment:
Shouldn't this file also import `01.css` and put the bellow section there?

Reply:
We can but not sure if we shoud? These are matrix-only related parameters, so they belong to matrix css I think. Or maybe you have a different approach on mind? Keep all relevant config params in 01.css maybe?